docs/src/tut/mkt/index.md

```diff
@@ -0,0 +1,1323 @@
+
+* Line 54 then `{!rsh} interact`s with the `Seller`'s `reportReady` `{!rsh} function`. 
+The `reportReady` takes in two arguments (i.e. the `Seller`'s `announcement` and `products`)
+
+* Line 55 concludes the `Seller`'s local step by calling `commit()`.
+
+Now, doing a `{!cmd} ./reach run` in the terminal will show some output.
+That concludes the `Seller`'s interaction.
+
+Next, type in the following code for the `Buyer`'s interaction:
+
+```reach
+load: /examples/mkt-2-basic/index.rsh
+range: 57 - 61
+```
+
+This is the `Buyer`'s local step
+
+* Line 57 calls the `{!rsh} only()` method on the `Buyer`.
+
+* Line 58 has the `Buyer` `{!rsh} interact` with the `shop` `{!rsh} function`. 
+The result of that interaction is also declassified. 
+
+* Line 60 publicizes the `decision` from line 58.
+
+* Line 61 concludes the `Buyer`'s steps with a `commit();`
+
+Finally, the result of the program is shown with the following code:
+
+```reach
+load: /examples/mkt-2-basic/index.rsh
+range: 63 - 63
+```
+
+* Line 63 executes the `showResult` `{!rsh} function` on both `{!rsh} Participants`.
+
+In the terminal, execute `{!cmd} ./reach run`.
+An output similar to what's printed below should display.
+
+```
+Welcome to the Market
+Contract info: {"type":"BigNumber","hex":"0x6d"}
+List of products for sale:
+1. Potatoes at 10 per bag.
+2. Carrots at 10 per bunch.
+3. Corn at 5 per ear.
+Buyer wants Potatoes
+Buyer agrees to buy 9 bags of Potatoes
+Seller agrees to sell 9 bags of Potatoes
+
+
+
+Welcome to the Market
+Contract info: {"type":"BigNumber","hex":"0x89"}
+List of products for sale:
+1. Potatoes at 10 per bag.
+2. Carrots at 10 per bunch.
+3. Corn at 5 per ear.
+Buyer wants Carrots
+Buyer agrees to buy 10 bunches of Carrots
+Seller agrees to sell 10 bunches of Carrots
+
+
+
+Welcome to the Market
+Contract info: {"type":"BigNumber","hex":"0x90"}
+List of products for sale:
+1. Potatoes at 10 per bag.
+2. Carrots at 10 per bunch.
+3. Corn at 5 per ear.
+Buyer wants Carrots
+Buyer agrees to buy 81 bunches of Carrots
+Seller agrees to sell 81 bunches of Carrots
+```
+
+:::note 
+`{!cmd} ./reach run` was executed 3 times to the 3 different outputs that you see above.
+
+You get one result each time that you execute `{!cmd} ./reach run`.
+:::
+
+This is a beautiful start. 
+You just created a simple DApp that makes a simple transaction happen.
+
+### {#tut-mkt-basic-Review} Review
+This section taught the basics of the Reach language and demonstrated how the final DApp to be built will work. You were able to learn that:
+
+* A `{!rsh} Participant` is a user.
+
+* An integer is represented by `{!rsh} UInt`. 
+
+* A string is represented by `{!rsh} Bytes(x)`.
+The integer (x) in braces defines how long the string may be.
+
+* A `{!rsh} function` in Reach is defined as `{!rsh} Fun([input], output)`.
+
+* `{!rsh} parseCurrency` `{!rsh} function` was provided by Reach to help convert regular currencies into currencies acceptable on a DApp.
```

Comment:
convert regular currencies into currencies acceptable on a DApp. <-- this needs some clarity, what is a "regular" currency and what is a "currency acceptable on a DApp"? I would not know what is intended except I already know what `parseCurrency` means.